pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=60", "wheel", "setuptools_scm[toml]>=8.0", 'oldest-supported-numpy']
+requires = ["setuptools>=60", "wheel", "setuptools_scm[toml]>=8.0", "numpy>=2.0"]
```

Comment:
Requiring numpy>=2.0 might be overkill as the proposed changes seem to be OK with numpy 1.x. So, I suggest undo this change.

Reply:
This is only the build requirements, and I think it's important to build the wheels using >=2.0, but for the user it's OK with >=1.25.3 (so that's what I set in install_requires). This is in line with the example here: https://numpy.org/doc/stable/dev/depending_on_numpy.html#numpy-2-0-specific-advice

Reply:
OK, sounds good to me!